app/src/main/kotlin/com/contentful/tea/kotlin/contentful/Contentful.kt

```diff
@@ -125,12 +127,12 @@ open class Contentful(
                     client
                         .fetch(CDAEntry::class.java)
                         .withContentType("course")
-                        .where("locale", parameter.locale)
+                        .where("locale", parameter.locale ?: DEFAULT_LOCALE)
```

Comment:
Why use a different syntax here than above `?:` versus `.or`

Reply:
`.or()` also checks if the string is empty (""). Elvis (?:) only checks for `null`. In this case if it is null, it has a different meaning then if it is empty. Therefore this should be different.